ticketing.md

```diff
@@ -0,0 +1,28 @@
+# Ticketing
+
+This team is responsible for making sure that PyCon's ticket system is working and that the money is properly accounted for. Because this team involves money, the treasurer of Python.Ph will be the de-facto leader of this committee.
+
+There should be a cut-off for tickets sold online. This is to give the entire PyCon team some breathing room before PyCon. It will be under the discretion of the PyCon head if tickets will still be sold at the event.
+
+During the conference, merchandise will be sold. As a general rule, no change will be given on anything sold at PyCon. The change will instead be considered as donation to Python.Ph. The exact nature of the donation will be determined beforehand, but on previous PyCons, it was decided that the donation will be used to fund local Python groups.
```

Comment:
> As a general rule, no change will be given on anything sold at PyCon. The change will instead be considered as donation to Python.Ph.

Not a lawyer but I'm not sure if this is 100% legal. It comes across as a fishy way to get donations.

If you are sure that there is no legal trouble in this, I suggest mentioning somewhere that you must put up prominent signs stating this fact, maybe at the merch booth itself. Some attendees may not be aware of this and might find it a bad experience.